zk_prover/src/circuits/merkle_sum_tree.rs

```diff
@@ -107,23 +115,26 @@ where
 /// * `poseidon_middle_config`: Configuration for the poseidon hash function with WIDTH = 2 and RATE = 1 and input length of 2 * (1 + N_ASSETS). Needed to perform hashings from the leaf to the root.
```

Comment:
The comment should likely be updated to `N_ASSETS + 2` instead of `2 * (1 + N_ASSETS)`

Reply:
Good catch! Thanks for that